src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -123,6 +123,55 @@ namespace UiaOperationAbstraction
             }
         }
 
+        template<class TryBody, class CatchBody>
+        void Try(TryBody&& tryBody, CatchBody&& catchBody) const
+        {
+            if (m_useRemoteApi)
+            {
+                m_remoteOperation.TryBlock(std::forward<TryBody>(tryBody), std::forward<CatchBody>(catchBody));
+            }
+            else
+            {
+                try
+                {
+                    tryBody();
+                }
+                catch(std::exception /*e*/)
+                {
+                    catchBody();
+                }
+            }
+        }
+
+        template<class TryBody>
+        void Try(TryBody&& tryBody) const
+        {
+            if (m_useRemoteApi)
+            {
+                m_remoteOperation.TryBlock(std::forward<TryBody>(tryBody));
+            }
+            else
+            {
+                try
+                {
+                    tryBody();
+                }
+                catch(std::exception /*e*/)

```

Comment:
Is this enough to be `std::exception` only?

Reply:
Good catch. I think this should probably be `catch (...)`.